src/VerifyExamples.elm

```diff
@@ -60,11 +60,13 @@ update msg model =
 
         CompileModule ( moduleName, fileText ) ->
             let
-                generatedTestFileText =
+                generatedTests =
                     Parser.parse fileText
-                        |> Compiler.compile moduleName
+                        |> List.concatMap (Compiler.compile moduleName)
             in
-                model ! [ writeFile ( moduleName, generatedTestFileText ) ]
+            ( model
+            , writeFiles generatedTests
```

Comment:
@eeue56 changed it to send everything to js and write these generatedTests in serial

Reply:
👍